nml/actions/action2var_variables.py

```diff
@@ -240,6 +249,26 @@ def signed_byte_parameter(name, args, pos, info):
     'other_veh_z_offset'  : {'var': 0x62, 'start': 24, 'size': 8, 'param_function':signed_byte_parameter, 'value_function':value_sign_extend},
 }
 
+varact2vars60x_trains = {
+    'tile_knows_rail_type'    : {'var': 0x63, 'start':  0, 'size': 1, 'param_function':vehicle_railtype},
```

Comment:
All existing properties/variables use `railtype` rather than `rail_type`.

Reply:
"all" is not true. But I copied from the minority :)